src/async.cpp

```diff
@@ -56,6 +56,30 @@ static int fuse_getattr(const char *path, struct stat *stbuf, struct fuse_file_i
 		return 0;
 	}
 	if(ret == 2) {
+		char f[32];
+		int fd;
+		ret = sscanf(path, "/%d/fdinfo/%d/%s", &pid, &fd, f);
```

Comment:
```suggestion
		char f[32] = {};
		int fd;
		ret = sscanf(path, "/%d/fdinfo/%d/%31s", &pid, &fd, f);
```

Reply:
Same for below usages (L117/123 and L129/130)

Reply:
> Am i correct that we are not managing /pid/fdinfo/fd/name reads in this PR?

@FedeDP Correct, we need another async event like `ASYNC_EVENT_FD_ENTRY_NAME` for that.
But we can do that in a separate PR imo :)
This can already be used to check if a threadinfo is missing some fds